test/suite/test_backup13.py

```diff
@@ -177,8 +124,7 @@ def test_backup13(self):
         bkup_c.close()
 
         # Make sure after a force stop we cannot access old backup info.
-        config = 'incremental=(src_id="ID1",this_id="ID3")'
-
+        config = 'incremental=(src_id="ID1",this_id="ID2")'
```

Comment:
I don't think ID3 exist, so i changed this, such that we can make sure that we are checking access rights for old backups let me know If I should change this back

Reply:
You're correct `ID3` does not exist. The `this_id` parameter is supposed to be new. Both `ID1` and `ID2` have been used. So while this config should return an error due and not find `ID1` if it fails or something is wrong, we want to use an unused ID.

Reply:
oh i see, will revert back thanks!